libs/sdk-ui-tests-e2e/cypress/tools/drillToModal.ts

```diff
@@ -76,4 +76,8 @@ export class DrillToModal {
             .click();
         return this;
     }
+
+    getTitleHeaderModal(title: string) {
+        this.getTitleElement().should("have.text", `${title}`);
```

Comment:
no need `${title}` just only `title` is enough.
this method is verified, no action `get`, it should be like `hasTitleHeader`